diff --git a/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/graal/snippets/StackOverflowCheckFeature.java b/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/graal/snippets/StackOverflowCheckFeature.java
index 225aace7140..eeb9f1243f8 100644
--- a/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/graal/snippets/StackOverflowCheckFeature.java
+++ b/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/graal/snippets/StackOverflowCheckFeature.java
@@ -68,7 +68,7 @@ public final class StackOverflowCheckFeature implements InternalFeature {
          */
         ListIterator<BasePhase<? super MidTierContext>> position = suites.getMidTier().findPhase(LoweringPhase.class);
         position.previous();
-        position.add(new InsertStackOverflowCheckPhase());
+        // position.add(new InsertStackOverflowCheckPhase());   
     }
 
     @Override
